diff --git a/PKGBUILD b/PKGBUILD
index 56d108d..d566882 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -4,8 +4,8 @@ _cudaarch=Common
 _pkgname=mxnet
 pkgname=('mxnet-git' 'mxnet-mkl-git' 'mxnet-cuda-git' 'mxnet-cuda-mkl-git')
 _pkgver=1.5.1
-pkgver=1.5.1.r10454.3d38dbde74
-pkgrel=2
+pkgver=1.5.1.r10461.18c9a69e2f
+pkgrel=1
 pkgdesc="A flexible and efficient library for deep learning"
 arch=('x86_64')
 url="http://mxnet.io/"
